paper/manuscript.tex

```diff
@@ -48,132 +51,165 @@
+
+The recommended way of using \refnx\ is from a \conda\ environment, which offers package, dependency and environment management \cite{conda}, using the pre-compiled distributables on the \refnx\ conda-forge channel. These distributables are made as part of the release process using the same CI web-services used to test the code. The matrix of distributables covers the major Python versions currently in use, across the macOS, Windows, and Linux operating systems. Alternatively the package can be installed from source, either directly from the git repository, or via \pip\ from the version uploaded to PyPI.\footnote{https://pypi.python.org/pypi/refnx} Building from source requires a C compiler and the \Cython\ and \NumPy\ packages to be installed; further dependencies should be installed to run the test suite to verify that compilation and installation was successful.
+
+% unless refnx has an absolute promise of permanent backwards compatibility, conda packages of the current version will not run old analysis code and so conda won't be usable for the reproducibility aspect.
+
+% with wheels for refnx, is conda any easier than 'pip install refnx'
+
+% FIXME: is only numpy required for building? more is required to run the test suite and people should do so?
+
+\refnx\ is released under the BSD permissive open source licence. In addition its dependencies are released under open source licences which means that use is free of cost to the end user and also the user is free to modify and improve this software.
 
- The recommended way of using \refnx\ is from a \conda\ environment (a package, dependency and environment manager for software), using the pre-compiled distributables on the \refnx\ conda-forge channel. These distributables are made as part of the release process using the same CI web-services used to test the code. The matrix of distributables cover the major Python versions currently in use, across the macOS/Windows/Linux operating systems. Alternatively the package can be installed from source, either directly from the git repository, or via \pip\ from the version uploaded to PyPI (https://pypi.python.org/pypi/refnx). Building from source requires a C compiler and the \Cython\ and \NumPy\ packages to be installed.
-  
-\refnx\ is released under the BSD permissive open source licence. In addition its dependencies are released under open source licences which mean that use is free of cost to the end user.
+%FIXME refnx doesn't actually have a BSD licence; it has a BSD-like licence
 
 \section{Comments on reproducibility of analyses}
-In order for a given scattering analysis to be fully reproducible by others a general set of conditions need to be met:
+
+In order for a given scattering analysis to be fully reproducible by others, a general set of conditions need to be met:
 \begin{itemize}
   \item the datasets used need to be deposited with a journal article, or be freely available.
+  %FIXME is that the raw data or the reduced data?
```

Comment:
at least the reduced data, but it would be good to have the raw data, and the code to reduce that as well, and the reduction steps.

Reply:
Are we restricting the scope of the paper to the analysis or do we bring in the reduction part too?